package.json

```diff
@@ -17,9 +17,6 @@
   "devDependencies": {
     "apigeetool": "^0.14.2",
     "license-checker": "^25.0.1",
-    "minimist": "^1.2.2",
-    "newman": "^4.5.7",
```

Comment:
Looks fine to me. Just question around newman lib removal. I didn't see any requirement to remove it. Assume that no longer in use. Is that correct?

Reply:
It's a legacy of the original repo this was copied from, we don't used it in our project